ARGUS/ARGUS_Transforms.py

```diff
@@ -162,6 +166,15 @@ def make_slices(smin, smax, _start, _end):
                 img = np.stack([outmean, outstd, outstdMin, outstdMax])
             else:
                 img = np.stack([outmean, outstd])
+            if self.include_img:
+                outrandframe = arr[self.R.randint(0,self.num_slices - 1)]
+                outrandframe = outrandframe.reshape((1,outrandframe.shape[0],outrandframe.shape[1]))
+                img = np.concatenate([outrandframe,img])
+            if self.mean_abs_pixel_diff:
+                outframediff = np.absolute(np.diff(arr,axis=self.axis))
+                outmeandiff = np.mean(outframediff,axis=self.axis)
+                outmeandiff = outmeandiff.reshape((1,outmeandiff.shape[0],outmeandiff.shape[1]))

```

Comment:
I don't think this is necessary.  I could be wrong, but see computation of outmean and outstd.